weather-app/src/main/java/rcgonzalezf/org/weather/location/LocationManager.java

```diff
@@ -87,13 +89,21 @@ public void disconnect() {
     }
   }
 
-  @VisibleForTesting void onEmptyLocation(BaseActivity baseActivity, View content) {
+  @VisibleForTesting void onEmptyLocation(final BaseActivity baseActivity, View content) {
     if (baseActivity != null && content != null) {
       Snackbar.make(content, baseActivity.getString(R.string.location_off_msg),
           Snackbar.LENGTH_SHORT).show();
+      //After 5 seconds the activity will be recharged
+        new Handler().postDelayed(new Runnable(){
```

Comment:
I am adding 5 seconds delay to refresh the activity looking for an available Internet connection. I am wondering if  there is another way to make a better user experience.